### ISSUE TITLE
Snap Deals always creates an unsealed copy even though "AlwaysKeepUnsealedCopy = false"

### ISSUE THREAD

**Issue by jamerduhgamer** [filecoin-project/lotus#9396]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [X] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [X] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus version
Daemon:  1.17.0+mainnet+git.2830429ad+api1.5.0
Local: lotus version 1.17.1+mainnet+git.8db6a939c

lotus-miner version
Daemon:  1.17.1+mainnet+git.8db6a939c+api1.5.0
Local: lotus-miner version 1.17.1+mainnet+git.8db6a939c
```

### Describe the Bug

When performing snap deals, an unsealed copy will always be created even though the "AlwaysKeepUnsealedCopy = false".

### Logging Information

```text
james@miner:~$ cat .lotusminer/config.toml | grep "Always"
  AlwaysKeepUnsealedCopy = false

james@miner:/mnt/nas/storage/unsealed$ du -h
512     ./fetching
9.8T    .
```

### Repo Steps

0. Check that your configuration on the lotus miner is correct.
1. Run 'lotus-miner sectors snap-up'
2. Send a deal, import and seal deal into the snapped up sector
3. See that an unsealed copy was created in the long term storage.

**Comment by jamerduhgamer** [comment]
Hi @arajasek, I just checked with our client and they confirmed they have the `FastRetrieval` flag set to False when they propose deals. Let me know if there are any logs I could provide to help identify the cause!

**Comment by arajasek** [comment]
@jamerduhgamer Thanks! @Reiers is helping dig up more logs for this, but I'll let you know if more help is needed --- this is perplexing :D

**Comment by Reiers** [comment]
Yeah - after some closer monitoring it still doesn't honor the flag.
More 🔍 is needed...

**Comment by donkabat** [comment]
Hi, here the same, but just on new sector. May I delete unsealed copies manually?

lotus-miner version 1.18.1+mainnet+git.e68b8e96c
boostd version 1.5.1-rc2+git.b5768c8

**Comment by TippyFlitsUK** [comment]
Are you using Lotus to propose deals or Boost Don?

**Comment by donkabat** [comment]
yes, 1.5.0

**Comment by TippyFlitsUK** [comment]
Yes, the fix for this is in Boost 1.15.1-rc2

**Comment by Reiers** [comment]
Remember that client need to use the `--fast-retrieval=false` when a deal is proposed.
Closing again ! Thanks for checking and confirming @TippyFlitsUK 💯
